### ISSUE TITLE
Docs [logs]: document usage of LOGFLARE_API_KEY

### ISSUE THREAD

**Issue by AdditionAddict** [supabase/supabase#13729]
# Improve documentation

In local development when going to view logs I get error:
```
{
  "code": 500,
  "message": "LOGFLARE_API_KEY env variables are not set"
}
```

## Link 

Add a link to the page which needs improvement (if relevant)

## Describe the problem

I presume this is `https://logflare.app/`? But nothing in the docs says this and 

I'm assuming I'm early given launch week but thought I'd open issue to track

Trying this directly I get to the "Send Logs to this Source" page but not sure what to do with this to integrate to local development.

Ideally we could log without 3rd party for local dev too.

## Describe the improvement

A quick tutorial document page would suffice

## Additional context

Add any other context or screenshots that help clarify your question.

![image](https://user-images.githubusercontent.com/48436581/232067998-371deef7-f751-4a48-b8d1-4a6f65c66a15.png)

**Comment by MildTomato** [comment]
@Ziinc tagged you, thought you might know most about self host logs

**Comment by Ziinc** [comment]
This is a bug, the tab should not be happening in cli since the setup has not been done. Been able to replicate. 

@AdditionAddict please follow the local logs setup [here](https://supabase.com/docs/guides/cli/local-development#enabling-local-logging)

**Comment by Ben1980** [comment]
Have the same error message with version 1.52.0. Maybe the fix is not yet integrated.

**Comment by sweatybridge** [comment]
Yup, you are right. I've just bumped the studio image in cli. The fix will be released in v1.52.1.

**Comment by Ziinc** [comment]
Thanks for handling @sweatybridge 🙏
